typescript/utils/src/types.ts

```diff
@@ -5,6 +5,7 @@ export enum ProtocolType {
   Ethereum = 'ethereum',
   Sealevel = 'sealevel',
   Cosmos = 'cosmos',
+  CosmosModule = 'cosmosnative',
```

Comment:
this one line of addition necessitated all the other additions in this PR

chose `CosmosModule` to avoid confusion/clash with the existing `CosmosNative` TokenStandard

Reply:
I'd have no problems with having the enum variant being called `CosmosNative` but happy to keep it as it is too, because in any case we would need to use the value as `ProtocolType. CosmosNative` \ `ProtocolType. CosmosModule`

Reply:
ok cool lets keep it CosmosNative then to make it easy for ctrl+f